docker/Dockerfile

```diff
@@ -174,6 +177,8 @@ RUN function comment() { :; }; \
       rm -f /usr/bin/c++; \
       echo "source /etc/cccl.bashrc" >> /etc/bashrc; \
     fi; \
+    curl -SsL ${SCCACHE_URL} | tar -C /usr/bin -zf - --wildcards --strip-components=1 -x */sccache; \
```

Comment:
We should use `${CURL} -L` to be consistent with the other `curl` commands used in this `Dockerfile`

Reply:
Will do.